### ISSUE TITLE
Burning is too weak to even bother

### ISSUE THREAD

**Issue by Marcloure** [mlenser/kryx-rpg-issues#684]
I know this was talked in the past, but discussing it with a player, I think burning is not worth its cost currently.

So, burn deals 1d6 at the **end** of each of the creature's turns, this means that the burn can be stopped before the damage happens.
- This means a fireball doesn't deal 6d6+1d6 and takes an action, but rather 6d6 + a small chance of dealing 1d6, plus an action. 
- In contrast, Vitriolic Sphere deals 6d6+3d6
- Horrid Wilting deals 8d6 at once.
- Freezing Sphere deals 6d6 + slow 1

You may argue that taking an action is worth 2d6 immediately, 3d6 on the next turn, or a slow 1, however:

- It doesn't need to be your action, it can be anyone's action. A familiar or a minion can put out the fire. 
- The DC is really low, considering that you need to roll a 6 or more remove burning 1, an you can expend 10 feet + 50% of your speed to roll with advantage. **This is fair**, since failing means it actually takes 2 actions.

Now, that would be fine, since you can say that if a creature has minions to help, you shouldn't be using burning spells. However, monsters should almost never put out the fire, since 1d6 of dot isn't worth their action.  Also:

- Casting at higher levels is bad, since you are expending a lot of mana for a damage that probably won't happen (the DC doesn't increase, so expending more mana to deal burning 4 makes no sense)
- Considering that a monster will survive roughly 3 turns, the fireball will deal in 3 turns the same 9d6 damage a vitrolic sphere deals in 2 turns if the target never bother to put out the fire
- Maybe burning 3 is worth their action, but then you are expending a lot of mana (fireball at 4) for 8d6 + 1 action when you could instead cast Horrid Wilting for 12d6 damage, or Freezing Sphere for 10d6 + Slow 3 (which staggers). If the monster decides to never put out the fire, it will still take 3 turns to deal only 2d6 more than Horrid Wilting does immediately

___

_**TL;DR:**_ burning is mostly ignorable by monsters, and if it stacks enough to become a nuisance, you are expending a lot of mana for the monster to easily remove the condition. You would be better off casting a cold spell at the same mana for more damage and stagger.

**Comment by mlenser** [comment]
You've said that burning is mostly ignorable, but you also assume it not being ignored via wording like "a small chance of dealing 1d6". If it's being ignored then the 1d6 is gauranteed. If it's not being ignored, then it costs an action to _possibly_ end.

> A familiar or a minion can put out the fire.

If a familiar/minion uses their action to possible end the burning condition, then that familiar/minion didn't attack or use Help which would give the "big bad" advantage. And again, the action to end burning is just a possibility.

> The DC is really low, considering that you need to roll a 6 or more remove burning 1

**Comment by mlenser** [comment]
You've said that burning is mostly ignorable, but you also assume it not being ignored via wording like "a small chance of dealing 1d6". If it's being ignored then the 1d6 is gauranteed. If it's not being ignored, then it costs an action to _possibly_ end.

> A familiar or a minion can put out the fire.

If a familiar/minion uses their action to possible end the burning condition, then that familiar/minion didn't attack or use Help which would give the "big bad" advantage. And again, the action to end burning is just a possibility.

> The DC is really low, considering that you need to roll a 6 or more remove burning 1

Hmm? The DC is 10. So they'd need to roll a 10 or higher (or have +4 dex to roll a 6 or higher)

_________________


> Considering that a monster will survive roughly 3 turns, the fireball will deal in 3 turns the same 9d6 damage a vitrolic sphere deals in 2 turns if the target never bother to put out the fire

I think this part is something to look at.

**Comment by Marcloure** [comment]
Also, I forgot to mention on augmenting that the burning damage isn't applicable for half damage on successful save. So casting fireball at 4 mana when the creature succeeds mean you deal only half of 8d6, while other spells deal half of 10d6 or 12d6

**Comment by Marcloure** [comment]
> On a failure by 4 or less, the burning level is reduced by 1.
Burning condition has "on a failure by 4 or less, the burning level is reduced by 1."

**Comment by mlenser** [comment]
> Also, I forgot to mention on augmenting that the burning damage isn't applicable for half damage on successful save. So casting fireball at 4 mana when the creature succeeds mean you deal only half of 8d6, while other spells deal half of 10d6 or 12d6

It's not meant to be.
![image](https://user-images.githubusercontent.com/235857/109979458-35707800-7cff-11eb-97d3-785d255e87e5.png)

**Comment by Marcloure** [comment]
Yeah, I know it's not meant to be. I'm just pointing out another weaknesses of burning that should make it less worthy than it currently is. If the miss of burning spells is weaker than the miss of other spells, it's expected that their hit should be stronger, but my other points show that isn't quite the truth.

**Comment by Marcloure** [comment]
About solutions for this issue, I think increasing the burn amount per mana is the way to go. Increasing the burn DC is just more reasons for the monster to not try to put out the fire, while the damage would still be lower or equal to direct damage spells.

**Comment by mlenser** [comment]
First step to to calculate why the sheet is overvaluing burning. That's because of the action cost.

**Comment by Marcloure** [comment]
Maybe the action cost is fair, but the sheet is assuming the monster will likely use its action. If fireball was 6d6 and -1 action for sure, it would be very good in relation to other AOE burst spells

**Comment by mlenser** [comment]
- **Monsters**
  - All monsters with burning buffed as the value of burning changes [#684](https://github.com/mlenser/kryx-rpg-issues/issues/684)
- **Rules**
  - **Conditions**
    - Burning DC now scales based on the burning level [#684](https://github.com/mlenser/kryx-rpg-issues/issues/684)
    - Extinguishing a burning creature that is larger than you makes the check be made with disadvantage [#684](https://github.com/mlenser/kryx-rpg-issues/issues/684)
- **Themes**
  - **Fire**
    - Most fire spells and concoctions buffed as the value of burning changed [#684](https://github.com/mlenser/kryx-rpg-issues/issues/684)
